src/appleseed/renderer/kernel/lighting/directlightingintegrator.cpp

```diff
@@ -155,42 +155,47 @@ void DirectLightingIntegrator::compute_outgoing_radiance_light_sampling_low_vari
         }
     }
 
-    // Add contributions from light-tree lights only.
+    // Add contributions from the light set.
     if (m_light_sampler.has_lightset())
     {
+        ShadingComponents ls_radiance(Spectrum::Illuminance);
+
         sampling_context.split_in_place(3, m_light_sample_count);
 
-        for (size_t i = 0; i < m_light_sample_count; ++i)
+        for (size_t i = 0, e = m_light_sample_count; i < e; ++i)
         {
-            // Sample light tree-compatible lights only.
+            // Sample the light set.
             LightSample sample;
             m_light_sampler.sample_lightset(
                 m_time,
                 sampling_context.next2<Vector3f>(),
                 m_shading_point,
                 sample);
-            
+
+            // Add the contribution of the chosen light.
             if (sample.m_triangle)
             {
                 add_emitting_triangle_sample_contribution(
                     sampling_context,
                     sample,
                     mis_heuristic,
                     outgoing,
-                    radiance);
+                    ls_radiance);
```

Comment:
That wouldn't be accurate, it's not the radiance of one particular light sample, but of _all_ light samples for the light set (hence the `ls_` prefix). Maybe `lightset_radiance`?